apps/client/src/pages/ConclusionPage.js

```diff
@@ -99,6 +99,68 @@ const ConclusionPage = ({ topic, checker }) => {
   };
   const currentDateTime = date.toLocaleDateString("nl-NL", options);
 
+  // Demo saveConclusion function
+  const saveConclusion = () => {
+    // Demo getRegistryAnswer
+    const getRegistryAnswer = (id) => {
+      // How do we get the registry answer?
+      return "true";
+    };
+
+    // Demo getQuestionAndAnswers
+    const getQuestionAndAnswers = () => {
+      return this.stack.reduce((acc, question) => {
+        acc.push({
+          ...question,
+          // id: does `question` contain `id`? (Yes)
+          // text: does `question` contain `text`?
+          // answer: does `question` contain `answer`? // key: `answer` or `userAnswer`? (Yes)
+          // message: why do we need `message` again?
+          // questionHash: why do we need `questionHash` again?
+          registryAnswer: getRegistryAnswer(question.id), // key: `registryAnswer` or `registerAnswer`
+        });
+        return acc;
+      }, []);
+    };
```

Comment:
I would write it like this, I think.

```
const getQuestionAndAnswers = () => this.stack.map((question) => ({
  // ...
}));
```

Reply:
I think it's even better to `concat()` with `getUpcomingQuestions()` to make it more future proof, to share halfway